legacy/src/app/controllers/intro.js

```diff
@@ -89,7 +89,7 @@ function IntroController(
   };
 
   // If intro has been completed redirect to machine list.
-  if ($window.CONFIG.completed_intro) {
+  if (!$window.CONFIG.completed_intro) {
```

Comment:
This was just to allow testing after completing the intro, there is a way to change the config to allow this too.